stock_location_position/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Syvain Van Hoof (Okia sprl) <sylvainvh@okia.be>
+# Copyright 2016-2019 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Location Position",
+    "version": "14.0.1.0.0",
+    "author": "BCIM, Okia, Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "summary": "Add coordinate attributes on stock location.",
+    "category": "Stock Management",
+    "depends": ["stock"],
+    "data": ["views/stock_location.xml"],
+    "installable": True,
+    "development_status": "Alpha",
```

Comment:
Is this still Alpha ?

Reply:
Hello @rousseldenis 

I am not sure the status of this module. Maybe we need to do more test to evaluate the module as stable ? Let me know

Thanks